.github/workflows/publish.yaml

```diff
@@ -72,11 +78,9 @@ jobs:
         with:
           author_name: "ersilia-bot"
           author_email: "ersilia-bot@users.noreply.github.com"
-          message: "update version [skip ci]"
+          message: "Update version and release date"
```

Comment:
We should keep the `[skip ci]` in the commit message, and it's advisable to never remove it whenever it is there in the message. The `[skip ci]` makes it so that we don't run pipelines again as it is really not needed to run them at this step.

Reply:
Thank you for explaining the implication to me. I am always happy to learn from you and implemented accordingly. @DhanshreeA